samples/ftd_configuration/static_route_entry.yml

```
@@ -10,6 +10,15 @@
           value: 192.168.4.0/24
           dnsResolution: IPV4_AND_IPV6
           type: networkobject
+    - name: Setup second destination network for static route
```

```suggestion
    - name: Setup the second destination network for static route
```
And let's add an empty line above.

---

fixed